utils/v2tofhirr4/mappings.bal

```diff
@@ -40,24 +40,27 @@ public function stringToHl7(string msg) returns hl7:Message|error {
 # + hl7 - HL7 message as a string or an hl7v2:Message  
 # + customMapper - Custom mapper implementation
 # + return - FHIR Bundle as a json
-public function v2ToFhir(string|hl7:Message hl7, SegmentToFhir? customMapper = ()) returns json|error {
+public isolated function v2ToFhir(string|hl7:Message hl7, SegmentToFhir? customMapper = ()) returns json|error {
```

Comment:
Shall we rename? `SegmentToFhir` --> `V2SegmentToFhirMapper`

Reply:
Fixed with 99c9318de23837d5a439bcb33a0d0a58a7035f8c